src/fabric/src/fabric_util.erl

```diff
@@ -23,6 +23,8 @@
 -export([validate_all_docs_args/2, validate_args/3]).
 -export([upgrade_mrargs/1]).
 -export([worker_ranges/1]).
+-export([do_recovery/0]).
```

Comment:
Let's avoid using `fabric_util` and use `fabric2_util` for consistency. However, in this case if we only check the config value once from one module, let's keep it in that module as a local function for now.

Wonder if `recovery` or `soft_deletion` is a better name... The discussion so far has used the `soft_deletion` term, maybe we should use that as the config value? The section should probably be `[fabric]` just to keep consistent with other FDB related settings.

Reply:
Ah if it is an already setting that's fine. It'd still move the check for it to the module were we are reading the setting (fabric2_fdb).

Reply:
Sure, and moved this to `fabric2_util`